src/cmds/scripts/pbs_postinstall.in

```diff
@@ -243,22 +232,42 @@ create_conf() {
 
 perform_checks() {
 	fail=0
-	if [ ${PBS_START_SERVER} != 0 -a ! -x ${PBS_EXEC}/sbin/pbs_server ] ;then
-		echo "*** Server does not exist!"
-		fail=1
-	fi
-	if [ ${PBS_START_SCHED} != 0 -a ! -x ${PBS_EXEC}/sbin/pbs_sched ] ;then
-		echo "*** Scheduler does not exist!"
-		fail=1
-	fi
-	if [ ${PBS_START_COMM} != 0 -a ! -x ${PBS_EXEC}/sbin/pbs_comm ] ;then
-		echo "*** Communication agent does not exist!"
-		fail=1
-	fi
-	if [ ${PBS_START_MOM} != 0 -a ! -x ${PBS_EXEC}/sbin/pbs_mom ] ;then
-		echo "*** MOM does not exist!"
-		fail=1
+	if command -v systemctl >/dev/null 2>&1; then
```

Comment:
It will work fine even we can remove the check for systemd and remove the check for START switches because during the installation by default it will be off i.e 0.